extensions/openshell-podman-gateway/src/manager/gateway-container-manager.ts

```
@@ -0,0 +1,318 @@
+  }
+
+  private async findExistingGatewayContainer(
+    endpoints: readonly EndpointConnection[],
+  ): Promise<{ dockerode: Dockerode; containerId: string; state: string; port: number } | undefined> {
+    for (const endpoint of endpoints) {
+      const containers = await endpoint.dockerode.listContainers({ all: true });
+      const existing = containers.find(c => c.Labels?.[GATEWAY_LABEL]);
+
+      if (existing) {
+        const portStr = existing.Labels?.[GATEWAY_PORT_LABEL];
+        const port = portStr ? parseInt(portStr, 10) : undefined;
```

Biome (2.5.1)

[error] Use Number.parseInt instead of the equivalent global.

(lint/style/useNumberNamespace)